### ISSUE TITLE
Transformer baseline-1M results

### ISSUE THREAD

**Issue by vince62s** [OpenNMT/OpenNMT-tf#45]
To keep track of my tests:
Baseline-1M En to Fr.

Test1: batch of 64 sentences
Multibleu on test set: 35.56 after 70K steps // 36.14 after 100K steps.
same on Newstest2014: 27.59 // 28.01

Test2: batch of 4096 tokens (which about more than twice the size of 64 sentences)
Multibleu on test set after 100K steps: 36.88
Newstest2014: 27.98

Comparison with T2T:
After 70K steps, on 64 sentence batch 34.21 // After 100K steps 34.76
After 70K steps on token batch 4096 37.44 // newstest2014 28.85

**Comment by guillaumekln** [comment]
What tokenization did you use?

**Comment by beichao1314** [comment]
Hi, @vince62s. What is your configuration about transformer? I always didn't get a better BLEU score as expectation. Thanks a lot!

**Comment by vince62s** [comment]
@beichao1314 
make sure you have adam_withnoam_decay
command line look like
python -m bin.main train --model config/models/transformer.py --config wmt14-ende.yml config/optim/adam_with_noam_decay.yml

where wmt14-ende.yml is about the same as in the sample config
the second one also, which make sure to use adam optimizer with noam type decay.

**Comment by beichao1314** [comment]
@vince62s Thanks a lot! 👍 
I will try it.

**Comment by guillaumekln** [comment]
@beichao1314 See the configuration file [transformer_1gpu.yml](https://github.com/OpenNMT/OpenNMT-tf/blob/master/config/transformer_1gpu.yml).

**Comment by minhcongnguyen1508** [comment]
what is your training data resource?
